app/models/description.rb

```diff
@@ -4,8 +4,5 @@ class Description < ActiveRecord::Base
   validates :description, presence: true
 
   # THIS IS WRONG, JUST A TEST!!!!
```

Comment:
take this comment away haha you took off the test, but not the comment and if we read it later it's gonna be confusing